scanners/nessus/nessus_none.py

```diff
@@ -90,7 +94,9 @@ def scan_id(self) -> int:
         return self._scan_id
 
     def setup(self):
-        logging.debug(f"Creating new scan named {self.cfg.scan.folder}/{self.cfg.scan.name}")
+        logging.debug(
```

Comment:
There's a few line length changes here probably done by your IDE which are unnecessary, the line length used in pre-commit and pyproject.toml is 120. If you could undo those it would be very much appreciated and make it clearer what changes are being made